admin/CMakeLists.txt

```diff
@@ -1,21 +1,15 @@
 # Copy template files into <build_dir>/local_init
-set(WORKER_CONFIG_DIR "${CMAKE_BINARY_DIR}/local_init")
-file(MAKE_DIRECTORY ${WORKER_CONFIG_DIR})
+file(MAKE_DIRECTORY ${LOCAL_INIT_DIR})
+
+# Configure and copy herbert_on into the build directory.
+# This directory is added to the Matlab path when CTest is run, so that
+# parallel workers can access herbert_on/worker scripts.
+# Note that LOCAL_INIT_DIR is defined in the top-level CMakeLists
+configure_file("herbert_on.m.template" "${LOCAL_INIT_DIR}/herbert_on.m")
 if(${BUILD_TESTS})
-    configure_file("worker_4tests.m.template" "${WORKER_CONFIG_DIR}/worker_4tests.m")
+    configure_file("worker_4tests.m.template" "${LOCAL_INIT_DIR}/worker_4tests.m")
 endif()
-configure_file("worker_v2.m.template" "${WORKER_CONFIG_DIR}/worker_v2.m")
-
-set(_path_regex "[a-zA-Z0-9/\\\\_\\-\\.: ]+")
```

Comment:
For the record in RegEx within ranges (`[]`) you don't need to escape (except `\` and `-` if it is non-terminal), the following RegEx should work just as well and be more readable:
`"[a-zA-Z0-9/_.: \\-]+"`
(N.b. the `-` is moved to the end to not specify a range)